pyro/nn/module.py

```diff
@@ -161,8 +164,11 @@ def _pyro_set_supermodule(self, name, context):
         self._pyro_context = context
         for key, value in self._modules.items():
             assert isinstance(value, PyroModule)
-            assert not value._pyro_context.used, \
+            assert not value._pyro_context.active, \
                 "submodule {} has executed outside of supermodule".format(name)
+            if value._pyro_context.used:
```

Comment:
Why is this needed?

Reply:
Hmm..sorry, none of this is needed. This was from a debugging session that I thought was reverted in the PR. Will remove.